schema-lending.graphql

```diff
@@ -529,8 +568,8 @@ type Market @entity {
   " Liquidation penalty (or the liquidation bonus for liquidators) as a percentage value. It is the penalty/bonus price on the collateral when liquidators purchase it as part of the liquidation of a loan that has passed the liquidation threshold "
   liquidationPenalty: BigDecimal!
 
-  " Token that need to be deposited to take a position in protocol "
-  inputToken: Token!
+  " Tokens that need to be deposited in this market to take a position in protocol (should be alphabetized) "
+  inputTokens: [Token!]!
 
   " Token that is minted to track ownership of position in protocol (e.g. aToken, cToken). Leave as null if doesn't exist "
   outputToken: Token
```

Comment:
I am not how common this is that it is worthwhile to change our schema to support it, but euler-finance has two outputTokens per market, eToken (asset) and dToken (debt). I am tracking dToken with a customized field '_dToken' in the euler refactor. If we want to supply this, then we'll have to change outputTokenSupply and outputTokenPriceUSD to an array too.

Reply:
resolved by adding `TokenBalances` type